java-accesscontextmanager/README.md

```diff
@@ -8,41 +8,32 @@ Java idiomatic client for [Identity Access Context Manager][product-docs].
 - [Product Documentation][product-docs]
 - [Client Library Documentation][javadocs]
 
-
 ## Quickstart
 
 
 If you are using Maven, add this to your pom.xml file:
 
-
 ```xml
 <dependency>
-  <groupId>com.google.cloud</groupId>
-  <artifactId>google-identity-accesscontextmanager</artifactId>
-  <version>1.4.5</version>
+  <groupId>com.google.api.grpc</groupId>
+  <artifactId>proto-google-identity-accesscontextmanager-v1</artifactId>
```

Comment:
Interesting. The split repo's README is wrong. 
![image](https://user-images.githubusercontent.com/28604/195629763-dfc7d944-aeb4-46b5-95cd-915db37b332b.png)